src/System.Windows.Forms/src/System/Windows/Forms/ToolStripItem.cs

```diff
@@ -407,6 +418,89 @@ public virtual Image BackgroundImage
             }
         }
 
+        /// <summary>
+        ///  Gets or sets the <see cref="System.Windows.Input.ICommand"/> object whose <see cref="System.Windows.Input.ICommand.Execute(object?)"/>
+        ///  method will be called when the ToolStripItem's <see cref="Click"/> event gets invoked and
+        ///  <see cref="System.Windows.Input.ICommand.CanExecute(object?)"/> indicates that the command can be executed.
+        /// </summary>
```

Comment:
I don't think the last bit is nessecary. Ditto for the ButtonBase class
```suggestion
        /// <summary>
        ///  Gets or sets the <see cref="System.Windows.Input.ICommand"/> object whose <see cref="System.Windows.Input.ICommand.Execute(object?)"/>
        ///  method will be called when <see cref="Click"/> event is invoked.
        /// </summary>
```

Reply:
Let's put that in the remarks then.